os/arch/arm/src/tiva/Make.defs

```diff
@@ -56,7 +56,7 @@ CMN_ASRCS  = up_saveusercontext.S up_fullcontextrestore.S up_switchcontext.S
 CMN_ASRCS += vfork.S setjmp.S
 
 CMN_CSRCS  = up_assert.c up_blocktask.c up_copyfullstate.c up_createstack.c
-CMN_CSRCS += up_mdelay.c up_udelay.c up_exit.c up_idle.c up_initialize.c
+CMN_CSRCS += up_mdelay.c up_udelay.c up_exit.c up_initialize.c
```

Comment:
keep these changes as part of separate commit saying  as current commit is for s5j.